build.gradle

```diff
@@ -180,7 +181,7 @@ dependencies {
     systemTestImplementation 'org.junit.jupiter:junit-jupiter:5.4.0'
 
     systemTestCompile('io.netty:netty-tcnative-boringssl-static:2.0.6.Final:' + osdetector.classifier)
-    systemTestCompile('org.microbean:microbean-helm:2.8.2.1.0.5') {
+    systemTestCompile('org.microbean:microbean-helm:2.12.3.0.0.1-SNAPSHOT') {
```

Comment:
does this snapshot have new feature not available in production version?

Reply:
I was having issues getting requirements to work, though if there are concerns I can try downgrading and see if I can sort things out. I was worried that since the library is expecting helm v 2.8 (from early 2018) and kubernetes 1.9 and our helm charts are written against 2.13 and 1.11, it'd be more likely that we'd run into compatibility issues that would be hard to track down.

Reply:
I was able to downgrade. The issues I had were fixed with changes unrelated to the version of the library I was using.